pkg/filter/dt/filter_mysql_prepare.go

```diff
@@ -29,13 +29,16 @@ import (
 	"github.com/cectc/dbpack/pkg/log"
 	"github.com/cectc/dbpack/pkg/misc"
 	"github.com/cectc/dbpack/pkg/proto"
+	"github.com/cectc/dbpack/pkg/tracing"
 	"github.com/cectc/dbpack/third_party/parser/ast"
 )
 
 func (f *_mysqlFilter) processBeforePrepareDelete(ctx context.Context, conn *driver.BackendConnection, stmt *proto.Stmt, deleteStmt *ast.DeleteStmt) error {
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_before_prepare_delete")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.BeforeImage`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "fetch_before_image")
```